README.md

```diff
@@ -174,8 +172,6 @@ Bookmarks
 
 [CS50's Introduction to Computer Science | edX](https://www.edx.org/course/cs50s-introduction-computer-science-harvardx-cs50x)
 
-[CS50](https://cs50.harvard.edu/2018/fall/)
```

Comment:
CS50 is a legendary course, it should definitely not be removed, as it is constantly updated each year.

Instead, we can add a link to some of the lectures or their youtube page which hosts the lectures each year:
2023 lectures: https://www.youtube.com/playlist?list=PLhQjrBD2T380F_inVRXMIHCqLaNUd7bN4
YT Page: https://www.youtube.com/channel/UCcabW7890RKJzL968QWEykA